src/index.html

```diff
@@ -1,13 +1,48 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <div class="card__photo">
+        <img
+          class="card__img"
+          src="../src/images/imac.jpeg"
+          alt="producte image"
+        >
+      </div>
+      <div class="card__name-product">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </div>
+      <div class="card__code-producte">
+        Код товара: 195434
+      </div>
+      <div class="card__rating">
+        <div class="card__stars">
+          <div class="card__star card__star_active"></div>
+          <div class="card__star card__star_active"></div>
+          <div class="card__star card__star_active"></div>
+          <div class="card__star card__star_active"></div>
+          <div class="card__star"></div>
+        </div>
+        <div class="card__count">Отзывов: 5</div>
+      </div>
+      <div class="card__cost">
+        <span class="card__price-text">Цена:</span>
+        <span class="card__price">69 999 грн</span>
+      </div>
+      <a
+        class="card__link"
+        href="#"
+        data-qa="hover"
+      >
+      КУПИТЬ
```

Comment:
```suggestion
        КУПИТЬ
```
add indent and use text-transform